thorlcr/slave/slavmain.cpp

```diff
@@ -756,9 +757,10 @@ class CKJService : public CSimpleInterfaceOf<IKJService>, implements IThreaded,
                 unsigned rowCount = getRowCount();
                 unsigned rowNum = 0;
                 unsigned rowStart = 0;
-                unsigned __int64 startSeeks = kmc->queryKeyManager()->querySeeks();
-                unsigned __int64 startScans = kmc->queryKeyManager()->queryScans();
-                unsigned __int64 startWildSeeks = kmc->queryKeyManager()->queryWildSeeks();
+                const CRuntimeStatisticCollection & stats = kmc->queryService().contextLogger.queryStats();
+                unsigned __int64 startSeeks = stats.getStatisticValue(StNumIndexSeeks);
+                unsigned __int64 startScans = stats.getStatisticValue(StNumIndexScans);
+                unsigned __int64 startWildSeeks = stats.getStatisticValue(StNumIndexWildSeeks);
```

Comment:
similar to hthorkey.cpp, line 207.
The other new jhtree stats will not be caught here.

Could open a new JIRA and revisit it separately.

I think should serialize the whole of 'stats' here, and deal with the delta the other side, by using updateStatsDeltaTo.

Reply:
I'll do this as separate issue: https://track.hpccsystems.com/browse/HPCC-29822